panel/layout/base.py

```diff
@@ -939,6 +939,12 @@ class Column(ListPanel):
         will update the scroll position of the Column. Setting to
         0 will scroll to the top.""")
 
+    scroll_index = param.Integer(default=None, bounds=(0, None), doc="""
```

Comment:
I think this should be implemented as a `scroll_to_index` method which sends an event to the frontend (e.g. like the HTML diff approach).